style.css

```diff
@@ -1,6 +1,6 @@
 * {
 	box-sizing:	border-box;
-	font-family: "Arial";
+	font-family: sans-serif;
 	background-color:#000000;
```

Comment:
this line still made it through despite [previous comment](https://github.com/CosminDarius1/Project-1/pull/4/files/9b19f6751d7d4360d9e409840660ca6a61e8ac91..c81be99f0d631b7b97f869e5147c7c6d831c24f5#r1262391461)?